AngularJS/src/services/httpService.js

```diff
@@ -0,0 +1,14 @@
+const _ = require('lodash');
```

Comment:
Use `import { tail } from 'lodash'` instead of the require. First, the `import` is used as a standard in ES6+ and the deconstruction is made to minimize the final bundle size and not to bring the whole lodash package unnecessary 😄